src/Humanizer/HeadingExtensions.cs

```diff
@@ -70,17 +71,61 @@ public static char ToHeadingArrow(this double heading)
         /// <summary>
         /// Returns a heading based on the short textual representation of the heading.
         /// </summary>
+        /// <param name="culture">The culture of the heading</param>
         /// <returns>The heading. -1 if the heading could not be parsed.</returns>
-        public static double FromAbbreviatedHeading(this string heading)
+        public static double FromAbbreviatedHeading(this string heading, CultureInfo culture = null)
```

Comment:
This should be an additional method/override, so we don't break existing people.

Reply:
Thanks! I think it should be fixed in 9d48857d9bc78cafdde18efcc0d8093674e8c6fd.